src/Cameras/Inputs/arcRotateCameraPointersInput.ts

```diff
@@ -139,7 +147,11 @@ export class ArcRotateCameraPointersInput extends BaseCameraPointersInput {
                 this.camera.inertialRadiusOffset +=
                     (pinchSquaredDistance - previousPinchSquaredDistance) * 0.001 *
                     this.camera.radius * this.pinchDeltaPercentage;
-            } else {
+            } else if (this.useNaturalPinchZoom) {

```

Comment:
This should override pinchDeltaPercentage, if set.  So put it as the first IF in this set of blocks.

Reply:
Good point, will fix.